website/blog/2023-01-18-release-0.11.md

```diff
@@ -0,0 +1,256 @@
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/6422176/213349477-979ee76f-1468-4663-8e49-c21cd6b276cf.mov' />
+
+### UI/UX Improvements 
+
+#### View Pods logs [#1122](https://github.com/containers/podman-desktop/pull/1122)
+
+The pods details view provides the ability to view the logs of each containers who may be running in a pod. Moreover, as it can also be hard to scan and identify from which container the outputs are coming, we appended the container name at the beginning of each output and differentiate each container with a different color. 
+
+![pods-logs](https://user-images.githubusercontent.com/6422176/211024673-eee9bad8-1b0c-4446-b8d1-97a226282c4d.png)
+
+#### Improved feedback when clicking on container list action icons [#1150](https://github.com/containers/podman-desktop/pull/1150) [#1161](https://github.com/containers/podman-desktop/pull/1161)
+
+When starting/stopping or deleting a container, a spinner is now displayed. In case of error, a message indicating that the action failed will also be better indicated. 
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/211531610-2347d302-4918-46ae-a5a2-c80fac0314f5.mp4' />
+
+For containers that exit immediately or short-lived containers, the feedback is also improved and include report of error now provide a better feedback to the user [#1161](https://github.com/containers/podman-desktop/pull/1161).
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/211831905-ebf596d5-efc8-4f55-8cb8-3f31655388b9.mp4' />
+
+
+#### Allows to change the default font size for the editor [#1160](https://github.com/containers/podman-desktop/pull/1160)
+
+The editors used is different screen of Podman Desktop, from the inspect screen, to container's outputs and Kubernetes YAML. The default font size is 10 pixels. It's now possible to adjust the font size to the one the one you prefer. This setting is persisted and will be used for all the editors of Podman Desktop and available from the preferences page (Settings -> Preferences).
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/211778161-130ff733-b2ca-4306-bea3-d031196c3b29.mp4' />
+
+#### Keep expanded state of pods when refreshing containers [#1042](https://github.com/containers/podman-desktop/pull/1042)
+
+When switching from different screens of the application or simply refreshing the list of containers, the expanded state of each item in the list is now persisted and will be properly restored.
+
+<ReactPlayer playing controls url='https://user-images.githubusercontent.com/436777/207864147-b68ea9bd-0ca9-42dc-882e-b8a705233749.mp4' />
+
+#### Click on the Pod name redirects to the Pod details page [#1159](https://github.com/containers/podman-desktop/pull/1159)
+
+The list of containers displays pods which may also be running as well. Now clicking on the pod name directly redirects to the Pod details page.
```

Comment:
"The list of containers displays pods which may also be running as well." -> "The list of containers also displays running pods."